crates/api_models/src/payments.rs

```diff
@@ -670,33 +679,44 @@ pub struct PaymentsResponse {
     pub error_message: Option<String>,
 }
 
-#[derive(Clone, Debug, serde::Deserialize)]
+#[derive(Clone, Debug, serde::Deserialize, ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct PaymentListConstraints {
+    /// The identifier for customer
     pub customer_id: Option<String>,
+    /// A cursor for use in pagination, fetch the next list after some object
     pub starting_after: Option<String>,
+    /// A cursor for use in pagination, fetch the previous list before some object
     pub ending_before: Option<String>,
+    /// limit on the number of objects to return
     #[serde(default = "default_limit")]
     pub limit: i64,
+    /// The time at which payment is created
     #[serde(default, with = "common_utils::custom_serde::iso8601::option")]
     pub created: Option<PrimitiveDateTime>,
+    /// Time less than the payment created time
     #[serde(default, with = "common_utils::custom_serde::iso8601::option")]
     #[serde(rename = "created.lt")]
     pub created_lt: Option<PrimitiveDateTime>,
+    /// Time greater than the payment created time
     #[serde(default, with = "common_utils::custom_serde::iso8601::option")]
     #[serde(rename = "created.gt")]
     pub created_gt: Option<PrimitiveDateTime>,
+    /// Time less than or equals to the payment created time
     #[serde(default, with = "common_utils::custom_serde::iso8601::option")]
     #[serde(rename = "created.lte")]
     pub created_lte: Option<PrimitiveDateTime>,
+    /// Time greater than or equals to the payment created time
     #[serde(default, with = "common_utils::custom_serde::iso8601::option")]
     #[serde(rename = "created.gte")]
     pub created_gte: Option<PrimitiveDateTime>,
 }
 
 #[derive(Clone, Debug, serde::Serialize)]
 pub struct PaymentListResponse {
+    // The size of the payment list
     pub size: usize,
+    // The list of payment response
     pub data: Vec<PaymentsResponse>,
```

Comment:
```suggestion
    /// The number of payments included in the list
    pub size: usize,
    /// The list of payments response objects
    pub data: Vec<PaymentsResponse>,
```

You had added regular comments. I'm assuming you meant to add doc comments instead.